package.json

```diff
@@ -64,6 +64,7 @@
     "@adonisjs/session": "6.4.0",
     "@adonisjs/shield": "7.1.1",
     "@adonisjs/view": "6.2.0",
+    "adonis5-jwt": "1.1.7",
```

Comment:
https://github.com/maxgalbu/adonis5-jwt hasn't had any commits for a year now. which means we will pull in some outdated dependencies eventually through them.

But I guess there is no alternative.

Reply:
yep... I dont think we do... The only alternative would be to refactor both Ferdium-App and Ferdium-Server afterwards - which would result in Breaking Changes on both sides for users that haven't been updating the client (they would be forced to update unless we keep a v1 API that works with jwt middleware).